PokeOneWeb.DataSync/ReadModelUpdate/Impl/ReadModelUpdateService.cs

```diff
@@ -100,7 +104,127 @@ public void UpdateReadModel(SpreadsheetImportReport importReport)
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                WriteIndented = true
+            };
+
+            //
+            // Entities
+            // 
+            Console.WriteLine("generating json files for entities");
+            ICollection<EntityTypeReadModel> entities = _entityTypeMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/entities.json", JsonSerializer.Serialize(entities, serializeOptions));
+
+            foreach (var entity in entities)
+            {
+                File.WriteAllText("resources/entities/" + entity.ResourceName + ".json", JsonSerializer.Serialize(entity, serializeOptions));
+            }
+
+            //
+            // itemstats
+            // 
+            Console.WriteLine("generating json files for itemstats");
+            ICollection<ItemStatBoostPokemonReadModel> itemStats = _itemStatBoostPokemonMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/itemstats.json", JsonSerializer.Serialize(itemStats, serializeOptions));
+
+            //
+            // learnable moves
+            //
+            Console.WriteLine("generating json files for learnable-moves");
+            ICollection<SimpleLearnableMoveReadModel> learnableMoves = _simpleLearnableMoveMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/learnable-moves.json", JsonSerializer.Serialize(learnableMoves, serializeOptions));
+
+            foreach (var learnableMove in learnableMoves)
+            {
+                File.WriteAllText("resources/learnable-moves/" + learnableMove.MoveResourceName + ".json", JsonSerializer.Serialize(learnableMove, serializeOptions));
+            }
+
+            //
+            // moves
+            //
+            Console.WriteLine("generating json files for moves");
+            ICollection<MoveReadModel> moves = _moveMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/moves.json", JsonSerializer.Serialize(moves, serializeOptions));
+            foreach (var move in moves)
+            {
+                File.WriteAllText("resources/moves/" + move.ResourceName + ".json", JsonSerializer.Serialize(move, serializeOptions));
+            }
+
+            //
+            // natures
+            //
+            Console.WriteLine("generating json files for natures");
+            ICollection<NatureReadModel> natures = _natureMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/natures.json", JsonSerializer.Serialize(natures, serializeOptions));
+
+
+            //
+            // varieties
+            //
+            Console.WriteLine("generating json files for varieties");
+            ICollection<PokemonVarietyReadModel> varieties = _pokemonVarietyMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/varieties.json", JsonSerializer.Serialize(varieties, serializeOptions));
```

Comment:
Do you write the full set of details about Pokémon here? No reduced list for the Pokédex page?

Reply:
correct